### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
New instrument cal csvs, plus 2 Papa 10 cruise-specific ADCP cal csvs

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Papa 10 AM Updates](https://trello.com/c/KVwjFuoC/197-papa-10-am-updates)

**Comment by rtravis-whoi** [comment]
[PYV Instrument Cal CSVs [DT-184]](https://trello.com/c/Eo546k32/184-pyv-instrument-cal-csvs-dt-184)

**Review by asmith-whoi — APPROVED** [review]
all ok

File: calibration/ADCPSL/CGINS-ADCPSL-22846__20230228.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+22846,CC_scale_factor1,0.45,constant; source_file ADCPS-L_WHLS75-1500-I_SN_22846_Configuration_Summary_2023-02-28.pdf
+22846,CC_scale_factor2,0.45,constant
+22846,CC_scale_factor3,0.45,constant
+22846,CC_scale_factor4,0.45,constant
+22846,CC_bin_size,0,in cm; return from vendor - not allocated yet
+22846,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+22846,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-22846__20230228.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-22846__20230513.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+22846,CC_scale_factor1,0.45,constant; GP03FLMA-00010
+22846,CC_scale_factor2,0.45,constant
+22846,CC_scale_factor3,0.45,constant
+22846,CC_scale_factor4,0.45,constant
+22846,CC_bin_size,1000,in cm
+22846,CC_dist_first_bin,1647,in cm
+22846,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-22846__20230513.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-23380__20230222.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+23380,CC_scale_factor1,0.45,constant; source_file ADCPS-L_WHLS75-1500-I_SN_23380_Configuration_Summary_2023-02-22.pdf
+23380,CC_scale_factor2,0.45,constant
+23380,CC_scale_factor3,0.45,constant
+23380,CC_scale_factor4,0.45,constant
+23380,CC_bin_size,0,in cm; return from vendor - not allocated yet
+23380,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+23380,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-23380__20230222.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-23381__20230322.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+23381,CC_scale_factor1,0.45,constant; source_file ADCPS-L_WHLS75-1500-I_SN_23381_Configuration_Summary_2023-03-22.pdf
+23381,CC_scale_factor2,0.45,constant
+23381,CC_scale_factor3,0.45,constant
+23381,CC_scale_factor4,0.45,constant
+23381,CC_bin_size,0,in cm; return from vendor - not allocated yet
+23381,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+23381,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-23381__20230322.csv:8]: looks good

File: calibration/ADCPTG/CGINS-ADCPTG-19151__20230121.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+19151,CC_scale_factor1,0.45,constant; source_file ADCPT-G_WHS150-1500-I_SN_19151_Configuration_Summary_2023-01-21.pdf
+19151,CC_scale_factor2,0.45,constant
+19151,CC_scale_factor3,0.45,constant
+19151,CC_scale_factor4,0.45,constant
+19151,CC_bin_size,0,in cm; return from vendor - not allocated yet
+19151,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+19151,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPTG/CGINS-ADCPTG-19151__20230121.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-23380__20230513.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+23380,CC_scale_factor1,0.45,constant; GP03FLMB-00010
+23380,CC_scale_factor2,0.45,constant
+23380,CC_scale_factor3,0.45,constant
+23380,CC_scale_factor4,0.45,constant
+23380,CC_bin_size,1000,in cm
+23380,CC_dist_first_bin,1647,in cm
+23380,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-23380__20230513.csv:8]: looks good

File: calibration/ADCPTG/CGINS-ADCPTG-24544__20230210.csv [hunk 7]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+24544,CC_scale_factor1,0.45,constant; source_file ADCPT-G_WHS150-1500-I_SN_24544_Configuration_Summary_2023-02-10.pdf
+24544,CC_scale_factor2,0.45,constant
+24544,CC_scale_factor3,0.45,constant
+24544,CC_scale_factor4,0.45,constant
+24544,CC_bin_size,0,in cm; return from vendor - not allocated yet
+24544,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+24544,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPTG/CGINS-ADCPTG-24544__20230210.csv:8]: looks good

File: calibration/PRESFB/CGINS-PRESFB-01386__20230127.csv [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+26P76848-1386,CC_pressure_offset_calibration_coefficient,-3.050000e-01,
+26P76848-1386,CC_slope_correction_factor,1,constant
+26P76848-1386,CC_t1,2.763804e+01,
+26P76848-1386,CC_t2,4.278750e-01,
+26P76848-1386,CC_t3,1.776240e+01,
+26P76848-1386,CC_t4,2.186445e+01,
+26P76848-1386,CC_u0,5.810283e+00,
+26P76848-1386,CC_y1,-3.944121e+03,
+26P76848-1386,CC_y2,-1.073763e+04,
+26P76848-1386,CC_y3,0.000000e+00,
```
Comment (cdobs) [inline calibration/PRESFB/CGINS-PRESFB-01386__20230127.csv:19]: looks good

**Review by cdobs — APPROVED** [review]
everything looks good

**Pull request merged**